Backend/src/main/java/com/onbank/api/controller/TransferController.java

```diff
@@ -23,5 +27,11 @@
         List<Transfer> transfer = transferService.getTransfers();
         return transfer.stream().map(TransferTransformer::convertToDto).collect(Collectors.toList());
     }
+
+    @PostMapping
+    public TransferDto setTransfer(@Valid @RequestBody CreateTransferDto transferDto){
```

Comment:
1) createTransfer - ta nazwa lepsza bo biznesowa.
2) jeśli coś tworzymy to raczej nie zwracamy wyniku. Można zwrócić void i dać adnotację @ResponseStatus(HttpStatus.OK) nad metodą. Tak samo możemy dodać tą adnotację nad getTransfers (ale tam już zwracamy listę, adnotacja dodatkowo)